practice-one/src/assets/styles/abstracts/variable.css

```diff
@@ -0,0 +1,17 @@
+:root {
+  --primary-color:  #EBCB80;
+  --secondary-color: #4b4b4b;
+  --base-color: #FFF;
+  /*-----------------------------------------*\
+  #
+\*-----------------------------------------*/
+  --base-font-size: 16px;
```

Comment:
Should put color as prefix, it should be something like --font-size-base. Check all, please